specification/keyvault/resource-manager/Microsoft.KeyVault/stable/2022-11-01/keysManagedHsm.json

```diff
@@ -0,0 +1,667 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2022-11-01",
+    "title": "KeyVaultManagementClient",
+    "description": "The Azure management API provides a RESTful set of web services that interact with Azure Key Vault."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.KeyVault/managedHSMs/{name}/keys/{keyName}": {
```

Comment:
>subscriptions [](http://example.com/codeflow?start=6&length=13)

A bunch of comments havent been addressed yet. Please either address them or provide a reason why they havent been addressed

Reply:
Addressed some of the comments and provided reasoning for the rest. Thanks for reviewing the PR.